radicle-httpd/src/api.rs

```diff
@@ -126,9 +128,11 @@ pub struct PaginationQuery {
 }
 
 mod project {
+    use nonempty::NonEmpty;
     use radicle::git::Oid;
```

Comment:
move `use serde::Serialize` to the top as well, and separate non-radicle and radicle imports with an empty line.